locales/it/index.yml

```diff
@@ -875,8 +875,7 @@ onboarding:
   shareData:
     title: Configura IO
   thankYouPage:
-    title: Configurazione terminata!
-    body: Ora puoi iniziare a usare IO
```

Comment:
Should we remove this string from the english (and german) translation too?

Reply:
do io